src/cuga/cli/knowledge_cmds.py

```
@@ -58,13 +58,13 @@ def knowledge_config_get(
         with _ur.urlopen(url, timeout=10) as resp:  # noqa: S310 — operator tool
             payload = _json.loads(resp.read().decode("utf-8"))
     except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)